gc/gc-iceberg-mock/src/main/java/org/projectnessie/gc/iceberg/mocks/MockManifestEntry.java

```diff
@@ -51,7 +50,7 @@ public Status status() {
 
   public abstract String filePath();
 
-  public abstract StructType partitionType();
+  public abstract Types.StructType partitionType();
```

Comment:
Why do we sometimes import the inner class name, and sometime qualify it with its parent class name?

In this particular case, the inner class name seems to be non-ambiguous, so why do we change it to be qualified by the parent type?

Reply:
as stated in the PR description we resolve `DifferentNameButSame` violations by doing the minimal amount of changes... so this file uses more `Types.StructType` than `StructType` right now, we we prefer to use the former for the fix.

if we can come up with a uniform convention on how to handle these things, we can also apply that kind of fix.

imo the first step is making things uniform on a per file basis, wdyt?

Reply:
IMHO, shorter type references (i.e. importing the most specific name possible)  are preferable over smaller PR diff.

I'm open to other opinions too.